app/User.php

```diff
@@ -496,4 +496,77 @@ public function hasPayPal(): bool
         // @phpstan-ignore-next-line
         return $this->subscribed('kanka') && $this->subscriptions()->first() && str_contains($this->subscriptions()->first()->stripe_price, 'paypal');
     }
+
+    public function upgradePrice(string $period, string $tier): string
```

Comment:
Needs to be in either SubscriptionService, or t's own new service for an even cleaner code